python/cudf/cudf/_libxx/nvtx.pyx

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cudf._libxx.lib cimport *
+from cudf._libxx.lib import *
+
+
+def range_push(name, color='green'):
```

Comment:
There may be a way to more cleanly define that these are strings as opposed to general Python objects, but we should type these either way:
```suggestion
def range_push(object name, object color='green'):
```